src/index.js

```diff
@@ -0,0 +1,64 @@
+import './styles/style.css';
+
+import { addLike, getLikes } from './modules/like.js';
+import { fetchAllMovies } from './modules/movieApi.js';
+import { countMovies } from './modules/Counters.js';
+
+const renderMovies = (movies, likes) => {
+  const moviesContainer = document.querySelector('.movies-container');
+  movies.forEach(
+    ({
+      id, name, image, language, description,
+    }, index) => {
+      const movieCard = `
+        <div class="container" id=${id}>
+          <img src=${image.medium}>
+          <span>${name}</span>
+          <span>Language: ${language}</span>
+          <span> Description: ${description}</span>
+          <button class="testing">
+            <i class="fa-solid fa-heart"></i>
+          </button>
+          <span class="likes">${likes[index].likes}</span>
+          <button id=${id} class="comment">Comments</button>
```

Comment:
- [ ] As specified in the project requirements, kindly make sure that when a user clicks on the comment button a pop-up menu shows up that allows the user to write their comment about that specific movie. The comment should be stored with the `involvement-api `.  See below for more details:
<img width="960" alt="Screenshot 2023-06-10 at 16 41 27" src="https://github.com/hagan199/JavascriptCapstone/assets/107192076/9870efe3-f958-4db6-8d52-d1fccc741b14">

![Comments](https://github.com/hagan199/JavascriptCapstone/assets/107192076/89d5afc6-c080-4eb7-b5ad-16d87d19593b)

Reply:
That what not part of student A task.